src/views/Dashboard/Customer/Home/RequestTest/Canceled/OrderCanceled.vue

```diff
@@ -0,0 +1,71 @@
+<template lang="pug">
+  .customer-order-canceled
+    .customer-order-canceled__title You have cancelled your order!
+      
+
+    template
+      v-row.customer-order-canceled__row
+        LabDetailCard(
+          :serviceDetail="selectedService"
+        )
+        PaymentDetailCard(
+          :serviceDetail="selectedService"
+          :orderDetail="detailOrder"
+        )
+
+  
+</template>
+<script>
+
+import { mapState } from "vuex"
+import LabDetailCard from "../LabDetailCard.vue"
+import PaymentDetailCard from "../PaymentDetailCard.vue"
+
+export default {
+  name: "OrderCanceled",
+
+  components: {
+    LabDetailCard,
+    PaymentDetailCard
+  },
+
+  mounted() {
+    if (!this.selectedService) {
+      this.$router.push({ name: "customer-payment-history" })
+    }
+  },
```

Comment:
please update this to `created` and update the condition to one line statement

Reply:
updated